news/2019-01-30-new-changes-to-star-rating-performance-points.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Changes to Star Rating and Performance Points"
+date: 2019-01-30 01:00:00 +0000
+---
+
+After many years of stagnancy, some changes are finally coming to the way Star Rating and Performance Points are calculated! Read on to find out more.
+
+![](temporary.banner)
+
+Hello!
+
+It is with great anticipation that I, Xexxar, am here to announce changes to both Performance Points (aka: pp) and the star rating system. 
```

Comment:
This reads rather weirdly, since it leaves a lot to imagination exactly what you're anticipating, considering you're in the process of doing the announcement already.

Maybe simply say something about this being a long time coming, or "with great excitement".

Reply:
Changed.